docs/RUNNING.md

```diff
@@ -0,0 +1,79 @@
+# Running
+
+## Table of Contents
+
+- [Running components automatically](#running-components-automatically)
+- [Environment Variables](#environment-variables)
+- [Troubleshooting](#troubleshooting)
+
+## Running Components Automatically
+
+### eq-author
+
+The following environment variables should be placed in a .env.[environment] file in the eq-author folder.
+| Name                   | Description                             | Required |
```

Comment:
I don't know why but this table doesn't seem to render correctly.

Reply:
Think the previous line to the table doesn't have double space at the end.